test/functional/android/desired_capabilities.py

```diff
@@ -27,7 +27,8 @@ def get_desired_capabilities(app):
         'platformName': 'Android',
         'deviceName': 'Android Emulator',
         'app': PATH('../../apps/' + app),
-        'newCommandTimeout': 240
+        'newCommandTimeout': 240,
+        'automationName': 'UIAutomator2'
     }
 
     return desired_caps
```

Comment:
Do we use newline at end of file as per PEP8?

Reply:
It passes the pep8 check locally and I didn't make any change at the end of this file just really the addition of UIAutomator2

Reply:
`python -m autopep8 -r --global-config .config-pep8 -d .` runs on CI. I've relaxed the check for `max-line-length`, only. Is it enough for pep8?
(If I missed something, I'd like to add the check on CI)

Reply:
@KazuCocoa I think the relaxed 120 max line length is pretty good